test/metabase/driver/postgres_test.clj

```diff
@@ -353,8 +353,13 @@
                                :type     :query
                                :query    {:source-table (u/the-id table)
                                           :aggregation  [[:count]]
-                                          :breakout     [[:field (u/the-id field) nil]]}})]
-            (is (= (str "SELECT (\"json_alias_test\".\"bob\"#>> ?::text[])::VARCHAR  "
+                                          :breakout     [[:field (u/the-id field)
+                                                          {:temporal-unit :month,
+                                                           :metabase.query-processor.util.add-alias-info/source-table (u/the-id table),
+                                                           :metabase.query-processor.util.add-alias-info/source-alias "dontwannaseethis",
+                                                           :metabase.query-processor.util.add-alias-info/desired-alias "dontwannaseethis",
+                                                           :metabase.query-processor.util.add-alias-info/position 1}]]}})]
+            (is (= (str "SELECT date_trunc('month', CAST((\"json_alias_test\".\"bob\"#>> ?::text[])::VARCHAR  AS timestamp)) "
                         "AS \"json_alias_test\", count(*) AS \"count\" FROM \"json_alias_test\" "
                         "GROUP BY \"json_alias_test\" ORDER BY \"json_alias_test\" ASC")
```

Comment:
I was expecting to see a test showing how the "order by" and "group by" clauses are handled differently.

Reply:
this actually sounds pretty important, now that i think about it- will do